SharedPackages/BrowserServicesKit/Sources/PixelKit/WidePixel/WidePixel.swift

```diff
@@ -28,6 +28,7 @@ public protocol WidePixelManaging {
     func startFlow<T: WidePixelData>(_ data: T)
     func updateFlow<T: WidePixelData>(_ data: T)
     func completeFlow<T: WidePixelData>(_ data: T, status: WidePixelStatus, onComplete: @escaping PixelKit.CompletionBlock)
+    func discardFlow<T: WidePixelData>(_ data: T)
```

Comment:
Added for the case where we're maintaining a wide pixel instance during a purchase flow, but it turns into a restoration flow - when that happens, we discard the wide pixel.